elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -64,6 +64,9 @@ def __init__(
         # `None` dimension. This is because they have different shapes in
         # different iterations.
         # `tf.Variable` requires initial value if shape has `None` dimension.
+        self._embedding_and_ids_graph = []
+
+    def init_for_graph_mode(self):
```

Comment:
Should we detect whether it's graph mode and then just run this initializing function implicitly so we don't have to call `init_for_graph_mode()` manually?

Reply:
Good idea. Done